eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -1,30 +1,44 @@
 param(
+  # Please use the RepoOwner/RepoName format: e.g. Azure/azure-sdk-for-java
   [Parameter(Mandatory = $true)]
-  $RepoOwner,
+  $Repo,
   # Use this if a pull request might have been opened from one repo against another.
   # E.g Pull request opened from azure-sdk/azure-sdk prBranch --> Azure/azure-sdk baseBranch
   $ForkRepoOwner,
   [Parameter(Mandatory = $true)]
-  $RepoName,
-  [Parameter(Mandatory = $true)]
-  $BranchPrefix,
+  $BranchPrefix,  
+  # Date format: e.g. Tuesday, April 12, 2022 1:36:02 PM. Allow to use other date format.
+  $LastCommitOlderThan,
   [Parameter(Mandatory = $true)]
   $AuthToken
 )
 
 . (Join-Path $PSScriptRoot common.ps1)
 
-LogDebug "Operating on Repo [ $RepoName ]"
+LogDebug "Operating on Repo [ $Repo ]"
+if (($Repo -split "/").Count -lt 2) {
+  LogError "The $Repo does not have right format. Please use RepoOwner/RepoName. E.g. Azure/azure-sdk-for-java. Your input: $Repo"
+  exit 1
+}
+
+$RepoOwner = ($Repo -split "/")[0]
+$RepoName = ($Repo -split "/")[1]
+
 try{
-  $branches = (Get-GitHubSourceReferences -RepoOwner $RepoOwner -RepoName $RepoName -Ref "heads/$BranchPrefix" -AuthToken $AuthToken).ref 
+  $responses = Get-GitHubSourceReferences -RepoOwner $RepoOwner -RepoName $RepoName -Ref "heads/$BranchPrefix" -AuthToken $AuthToken
```

Comment:
If this is the only reason we are calling split above we can add a new parameter with the full name so that we can avoid the unnecessary parsing.